native_client/dotnet/DeepSpeechClient/Models/DeepSpeechStream.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace DeepSpeechClient.Models
```

Comment:
Can you ensure this new class is properly reflected in `doc/` ?

Reply:
Will do.

Reply:
Should be OK now.